src/Sulu/Bundle/AdminBundle/Resources/js/containers/Link/overlays/LinkTypeOverlay.js

```diff
@@ -57,6 +59,12 @@ export default class LinkTypeOverlay extends React.Component<LinkTypeOverlayProp
                         />
                     </Form.Field>
 
+                    {onQueryChange &&
```

Comment:
would be nice to add a test case for this similar to https://github.com/sulu/sulu/blob/b45f91ff36b468d200b254f894368cb0c3380671/src/Sulu/Bundle/AdminBundle/Resources/js/containers/Link/tests/overlays/LinkTypeOverlay.test.js#L51-L74